apps/pkey.c

```diff
@@ -258,6 +258,8 @@ int pkey_main(int argc, char **argv)
                            ERR_reason_error_string(err));
                 ERR_get_error(); /* remove err from error stack */
             }
+            EVP_PKEY_CTX_free(ctx);
```

Comment:
It is arguably cleaner to move the scope of ctx to the top level.. and then just do the free after label end:

Reply:
Done: 24f10b9b611067ce62342f2c6fa5d877a669519b